shop/src/Components/FeatureCard/index.jsx

```diff
@@ -1,15 +1,27 @@
 import "./FeatureCard.scss";
-const IMG =
-  "https://demo.evershop.io/assets/catalog/1034/3600/plv7632-Green-list.png";
-const FeatureCard = () => {
+import { dolorFormat } from "../../util/dolarFormat";
+const FeatureCard = ({ shoeItems }) => {
+
   return (
-    <div className="featureCard">
-      <img className="featureImg" src={IMG} alt="ImageOnCard" />
-      <div className="featureTextContainer">
-        <p className="featurePrimaryText">Nike air zoom pegasus 35</p>
-        <p className="featurePrimaryText">$411.00</p>
-      </div>
-    </div>
+    <>
+      {shoeItems.map((shoe, i) => {
```

Comment:
1. `shoesItems`, shoes with "s"
2. if the `shoesItems` is empty this component will ☠️